packages/atlas/src/views/studio/VideoWorkspace/VideoWorkspace.hooks.ts

```diff
@@ -34,6 +37,10 @@ export const useHandleVideoWorkspaceSubmit = () => {
   const { joystream, proxyCallback } = useJoystream()
   const startFileUpload = useStartFileUpload()
   const { channelId, memberId } = useAuthorizedUser()
+  const { data: channelVideosCount } = useGetVideoCountQuery({
+    skip: true,
```

Comment:
Would it change later or should it be smth like `!channelId`?

Reply:
Yeah, it should be `!channelId || !atlasConfig.general.appId`